vixos/template_nix.py

```diff
@@ -24,7 +28,8 @@ def generate_nix_user(package: str, executable: str) -> str:
   imports = [
     <nixpkgs/nixos/modules/virtualisation/qemu-vm.nix>
     ./managed.nix
-    ../local.nix
+    ./local.nix
+    ./global.nix
```

Comment:
global.nix should be in the parent directory, right?

Reply:
Fixed.